.github/workflows/build-pr.yml

```diff
@@ -8,13 +8,18 @@ on:
 jobs:
   run:
     runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        go: ['1.18', '1.19', '1.20']
```

Comment:
I'm always confused by the formats this action supports; is `1.18` equivalent to `1.18.0` or should it be `1.18.x`?

Reply:
`1.18` should pull the latest; however, don't mind updating to `1.18.x` just to be more clear.